var/spack/repos/builtin/packages/py-urllib3/package.py

```diff
@@ -13,6 +13,12 @@ class PyUrllib3(PythonPackage):
     homepage = "https://urllib3.readthedocs.io/"
     pypi = "urllib3/urllib3-1.25.6.tar.gz"
 
+    version("2.0.3", sha256="bee28b5e56addb8226c96f7f13ac28cb4c301dd5ea8a6ca179c0b9835e032825")
+    version(
+        "1.26.16",
+        sha256="8f135f6502756bde6b2a9b28989df5fbe87c9970cecaa69041edcce7f0589b14",
+        preferred=True,
+    )
```

Comment:
No more preference for 1.26 series.
```suggestion
    version("1.26.16", sha256="8f135f6502756bde6b2a9b28989df5fbe87c9970cecaa69041edcce7f0589b14")
```